packages/terra-clinical-data-grid/src/terra-dev-site/doc/clinical-data-grid/DatagridExamples.2/Datagrid.scss

```diff
@@ -0,0 +1,20 @@
+:local {
```

Comment:
@yuderekyu @bjankord shouldn't these sass files be named `name.module.scss`?

Reply:
Left a comment on a similar PR in terra-core for this: https://github.com/cerner/terra-core/pull/2465#discussion_r295826939

Reply:
Renamed name.scss to name.module.scss.